specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -271,6 +402,171 @@
         }
       }
     },
+    "MultiLanguageConversationAnalysisInput": {
+      "properties": {
+        "conversations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Conversation"
+          }
+        }
+      }
+    },
+    "Conversation": {
+      "required": [
+        "id",
+        "language",
+        "modality"
+      ],
+      "properties": {
+        "id": {
+          "description": "Identifier for the conversation",
+          "type": "string"
+        },
+        "language": {
+          "description": "Default languages of the conversation item in BCP-47 format",
```

Comment:
1. I would remove the word "Default". Nothing here is default or not.
2. Should be singular "language". Not plural.

Reply:
Should be singular. Regarding the default language Mikael and I discussed a scenario where the entire conversation will have a language which is default for the conversation. However, one turn can have a different language so that will be at the item level. So that's the reason to mention the language as default in the conversation level..

Reply:
Done.